src/main/java/org/carlspring/strongbox/janusgraph/cassandra/CassandraShutdown.java

```diff
@@ -0,0 +1,38 @@
+package org.carlspring.strongbox.janusgraph.cassandra;
+
+import javax.annotation.PreDestroy;
+import javax.inject.Inject;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import org.apache.cassandra.service.CassandraDaemon;
+import org.apache.cassandra.service.StorageService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author ankit.tomar
+ */
+@Configuration
```

Comment:
@anki2189  can you please make it as a bean in `CassandraEmbeddedConfig` ?

Reply:
Done.